nft_ingester/src/program_transformers/token_metadata/v1_asset.rs

```diff
@@ -311,38 +311,41 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
         "{} WHERE excluded.slot_updated > asset_authority.slot_updated",
         query.sql
     );
-    txn.execute(query).await?;
+    txn.execute(query)
+        .await
+        .map_err(|db_err| IngesterError::AssetIndexError(db_err.to_string()))?;
     if let Some(c) = &metadata.collection {
-        if c.verified {
-            let model = asset_grouping::ActiveModel {
-                asset_id: Set(id.to_vec()),
-                group_key: Set("collection".to_string()),
-                group_value: Set(Some(c.key.to_string())),
-                seq: Set(Some(0)),
-                slot_updated: Set(Some(slot_i)),
-                ..Default::default()
-            };
-            let mut query = asset_grouping::Entity::insert(model)
-                .on_conflict(
-                    OnConflict::columns([
-                        asset_grouping::Column::AssetId,
-                        asset_grouping::Column::GroupKey,
-                    ])
-                    .update_columns([
-                        asset_grouping::Column::GroupKey,
-                        asset_grouping::Column::GroupValue,
-                        asset_grouping::Column::Seq,
-                        asset_grouping::Column::SlotUpdated,
-                    ])
-                    .to_owned(),
-                )
-                .build(DbBackend::Postgres);
-            query.sql = format!(
-                    "{} WHERE excluded.slot_updated > asset_grouping.slot_updated AND excluded.seq >= asset_grouping.seq",
-                    query.sql
-                );
-            txn.execute(query).await?;
-        }
+        let model = asset_grouping::ActiveModel {
+            asset_id: Set(id.to_vec()),
+            group_key: Set("collection".to_string()),
+            group_value: Set(Some(c.key.to_string())),
+            verified: Set(Some(c.verified)),
+            seq: Set(None),
```

Comment:
if there is a conflict we update seq to None instead of 0? are those equivalent?

Reply:
They are equivalent, yes. The API will default None to 0.